oeps/oep-0026/xapi-realtime-events.rst

```diff
@@ -78,11 +78,11 @@ The **Verb** in xAPI is a past tensed value, identified by a URI from the `xAPI
 
 The registry is automatically created from multiple profiles. For now, we will limit ourselves to only URIs prefixed by the following domains, in the following priority order (in case of conflicting names):
 
-* http://adlnet.gov
-* http://w3id.org
-* http://id.tincanapi.com
+* ``http://adlnet.gov``
+* ``http://w3id.org``
+* ``http://id.tincanapi.com``
```

Comment:
These were working links. See https://open-edx-proposals.readthedocs.io/en/latest/oep-0026/xapi-realtime-events.html#xapi-verb
What would the tip be for doing this and passing tests? Does the following work?
```
`http://adlnet.gov`__
```
Or would it need to be?:
```
`http://adlnet.gov <http://adlnet.gov>`__
```
Other?

Reply:
They were showing up as links before but were dead, they shouldn't be links because they're actually referring to xapi event schema URIs and so aren't all necessarily working links.